include/seqan3/alphabet/nucleotide/dna4.hpp

```diff
@@ -256,6 +329,9 @@ struct dna4
         }()
     };
 
+    //!\brief The complement table.
+    static const std::array<dna4, value_size> complement_table;
```

Comment:
No, this required, unfortunately, because declaring it constexpr forces us to define inline which doesn't work, because we using types (dna4 itself) which are not yet defined.
See also:
https://stackoverflow.com/questions/11928089/static-constexpr-member-of-same-type-as-class-being-defined/12558623#12558623